Packs/PerimeterX/Integrations/BotDefender/BotDefender.yml

```diff
@@ -0,0 +1,103 @@
+category: Data Enrichment & Threat Intelligence
```

Comment:
You should remove all the params that are related to fetching incidents, as your integration does not support the fetch incidents command.

Reply:
Packs/PerimeterX/Integrations/BotDefender/BotDefender.yml: [IN129] - You've removed integration parameters, the removed parameters are '{'max_fetch', 'first_fetch', 'incidentType', 'isFetch'}'

I get this linter errors on commit if I remove